### PR TITLE
Set the default transformer for GPS column relationship

### LINKED ISSUE #1749: Set the default transformer for GPS column relationship
### Problem Description

_As a user, when I set a gps column relationship, I'm expecting the columns to be transformed with the MetroAreaAnonymizer if I have access to it and AnonymizedFaker otherwise_

### Expected behavior
Update the following dict inside the data processor to set the `MetroAreaAnonymizer` as default transformer for the gps relationship:
https://github.com/sdv-dev/SDV/blob/5ef78d32d3b89ea07ea4a330bc6588de19a14f55/sdv/data_processing/data_processor.py#L70

Also update the metadata validation similarly to what is done for the address relationship:
https://github.com/sdv-dev/SDV/blob/5ef78d32d3b89ea07ea4a330bc6588de19a14f55/sdv/metadata/validation.py#L22
### Additional context

The logic is the same as the address relationship, so it can be use as example.

### PR DESCRIPTION
**Pull request by R-Palazzo**
CU-86az4a0dz
Resolve #1749

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86az4a0dz SDV - Set the default transformer for GPS column relationship #1749](https://app.clickup.com/t/86az4a0dz)

File: sdv/metadata/validation.py [hunk 1]
```diff
@@ -23,13 +23,46 @@ def validate_address_sdtypes(columns_to_sdtypes):
...
+def _check_import_gps_transformers():
+    """Check that the gps transformers can be imported."""
+    error_message = (
+        'You must have SDV Enterprise with the gps add-on to use the gps features'
+    )
+    if not hasattr(rdt.transformers, 'gps'):
+        raise ImportError(error_message)
+
+    has_randomlocationgenerator = hasattr(rdt.transformers.gps, 'RandomLocationGenerator')
+    has_metroaeraanonymizer = hasattr(rdt.transformers.gps, 'MetroAreaAnonymizer')
```
Comment (amontanez24) [inline sdv/metadata/validation.py:48]: typo: metroaera -> metroarea

File: sdv/metadata/validation.py [hunk 2]
```diff
@@ -23,13 +23,46 @@ def validate_address_sdtypes(columns_to_sdtypes):
...
+    has_gpsnoiser = hasattr(rdt.transformers.gps, 'GPSNoiser')
+    if not has_randomlocationgenerator or not has_metroaeraanonymizer or not has_gpsnoiser:
+        raise ImportError(error_message)
+
+
+def validate_gps_sdtypes(columns_to_sdtypes):
```
Comment (amontanez24) [inline sdv/metadata/validation.py:54]: where is this being called?

Comment (R-Palazzo) [reply sdv/metadata/validation.py:54]: This is put into a dict here:
https://github.com/sdv-dev/SDV/blob/aca634f82ca5e39625487d3667ff5ff099903711/sdv/metadata/single_table.py#L109

And get called here:
https://github.com/sdv-dev/SDV/blob/aca634f82ca5e39625487d3667ff5ff099903711/sdv/metadata/single_table.py#L708

**Review by fealho — APPROVED** [review]

**Review by amontanez24 — APPROVED** [review]
LGTM!

**Pull request merged**